packages/tabs/tab.md

```diff
@@ -35,15 +35,21 @@ The `<sp-tab>` element consists of a label and an optional icon.
 
 ### Options
 
-#### Without a label
+#### Icon only
+
+For tabs that have an icon and no visible label, the icon should have a `label`. Icons should not be used just as decoration. If the tab item does not have a visible label, it must still have a tooltip to disclose the label.
 
 ```html
 <sp-tab value="complete">
-    <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
+    <sp-icon-checkmark
+        slot="icon"
+        label="Checking your work"
+        aria-label="Tab w/ checkmark"
```

Comment:
I'd have to check with Nikki again, but would we want the `label` and `aria-label` to be the same, that way screen reader users get the same experience & information? 

When I tested this in Safari, VoiceOver also announces: "Tab w slash checkmark" instead of what this was probably going for "Tab with checkmark."